ts/sagas/messages/messages.ts

```diff
@@ -31,26 +31,34 @@ export function* loadMessage(
   if (cachedMessage !== undefined && pot.isSome(cachedMessage.message)) {
     return right(cachedMessage);
   }
+  try {
+    // Fetch the message from the Backend
+    const maybeMessage: SagaCallReturnType<typeof fetchMessage> = yield call(
+      fetchMessage,
+      getMessage,
+      meta
+    );
 
-  // Fetch the message from the Backend
-  const maybeMessage: SagaCallReturnType<typeof fetchMessage> = yield call(
-    fetchMessage,
-    getMessage,
-    meta
-  );
-
-  if (maybeMessage.isLeft()) {
+    if (maybeMessage.isLeft()) {
+      yield put(
+        loadMessageAction.failure({
+          id: meta.id,
+          error: maybeMessage.value
+        })
+      );
```

Comment:
could it be just `throw(maybeMessage.value)` to reduce repetition?

Reply:
update within d932cb5617eba8ca99451ec35c20f3234c9c04ad